src/hexkit/providers/s3/testutils.py

```diff
@@ -61,21 +61,21 @@ class FileObject(BaseModel):
     file_path: Path
     bucket_id: str
     object_id: str
-    content: bytes = b"will be overwritten"
-    md5: str = "will be overwritten"
-
-    # pylint: disable=no-self-argument
-    @validator("content", always=True)
-    def read_content(cls, _, values):  # noqa: N805
-        """Read in the file content."""
-        with open(values["file_path"], "rb") as file:
+
+    @computed_field  # type: ignore [misc]
+    @property
+    def content(self) -> bytes:
+        """Extract the content from the file at the provided path"""
+        if not self.file_path:
+            return b"will be overwritten"
```

Comment:
Wouldn't it better better to raise an error in this case?

Reply:
That's not a bad idea. changed in the commit below.